docs/3_finetuner_in_action/3_2_image_to_image.md

```diff
@@ -1,3 +1,117 @@
+
+## Login to Finetuner
+As explained in the [Login to Jina ecosystem](../2_step_by_step/2_3_login_to_jina_ecosystem.md) section, first we need to login to Finetuner:
+```python
+import finetuner
+finetuner.login()
+```
+
+## Choosing the model
+Now let's see what backbone models we can use. We can see available models either in [the docs](../2_step_by_step/2_5_choose_back_bone.md) or by calling:
+
+```python
+finetuner.describe_models()
+```
+For this example, we're gonna go with `resnet50`.
+
+## Creating a fine-tuning job
+You can easily start a fine-tuning run with `finetuner.fit`. With Finetuner you can create several fine-tuning jobs which will run in parallel.
+Let's use this advantage now and create two runs: in the first run we'll freeze the weights and train an additional `MLP` layer, and in the second one we'll fine-tune the whole model. 
+
+```python
+from finetuner.callbacks import BestModelCheckpoint
+
+run1 = finetuner.fit(
+        model='resnet50',
+        run_name='resnet-ttl-1',
+        description='fine-tune the whole model.',
+        train_data='resnet-ttl-train-data',
+        eval_data='resnet-ttl-eval-data',
+        loss='TripletMarginLoss',
+        callbacks=[BestModelCheckpoint()],
+        epochs=6,
+        freeze=False,
+    )
+
+run2 = finetuner.fit(
+        model='resnet50',
+        run_name='resnet-ttl-2',
+        description='freeze all weights, and only fine-tune the additional MLP layer.',
+        train_data='resnet-ttl-train-data',
+        eval_data='resnet-ttl-eval-data',
+        loss='TripletMarginLoss',
+        callbacks=[BestModelCheckpoint()],
+        epochs=6,
+        freeze=True,
+    )
+```
+Now, let's understand what this code means. 
+```{admonition} finetuner.fit parameters
+:class: tip
+The only mandatory parameters are `model` and `train_data`. We provide default values for others. Here is the [full list of the parameters](../../api/finetuner/#finetuner.fit). 
+```
+As you can see, we have to provide the `model` which we picked before. We also set `run_name` and `description`, which are optional,
+but recommended in order to retrieve your run easily and have some context about it. Furthermore, we had to provide `train_data` and `eval_data`. As you can see,
+we used the names of the `DocumentArray`s that are already on Hubble, but we could also pass a `DocumentArray` object itself, which will be automatically uploaded to Hubble. As stated before, we want to use the `TripletLoss`, and that's what `loss='TripletMarginLoss'` corresponds to.
+We used `BestModelCheckpoint` callback and set `epochs` to 6. The only difference between these two runs comes with the parameter `freeze`. As explained in the `description` field of each run, `freeze=True` will freeze the existing weights, and train only the additional `MLP` layer, while `freeze=False` will fine-tune the whole model.
```

Comment:
Do you need to specify the size of the MLP layer that should be added? Or does `freeze=True` suffice?

Reply:
I think you're right. `output_dim` needs to be specified. I'll check it though.